.github/workflows/build-images-and-create-deployment.yml

```diff
@@ -7,6 +7,10 @@ on:
       - staging
       - prod
 
+# add a concurrency group to prevent multiple builds from running at the same time
+concurrency:
```

Comment:
Is this a new feature? I love it! We should add something like this for rdev created for a PR.

Reply:
@Bento007 - I don't know if it is a new feature but this was suggested by @atolopko-czi when I brought up the issue of concurrency and order of operations:

https://github.com/chanzuckerberg/single-cell-data-portal/issues/5461#issuecomment-1740899948